packages/coding-standard/src/Rules/CheckConstantStringValueFormatRule.php

```diff
@@ -46,7 +46,12 @@ public function process(Node $node, Scope $scope): array
         }
 
         foreach ($consts as $const) {
-            if ($const->name->toString() !== 'ERROR_MESSAGE' && ! $const->value instanceof String_) {
+            $constName = $const->name->toString();
+            if ($constName === 'ERROR_MESSAGE' || Strings::endWith($constName, '_REGEX')) {
+                continue;
+            }
```

Comment:
:+1: 

What about extracting these to a parameter? E.g.

```php
/** 
 * @var string[]
 */
private const ALLOWED_CONST_NAMES = [
    'ERROR_MESSAGE',
    '*_REGEXT',
];
```

Then skipping with `Symplify\PackageBuilder\Matcher\ArrayStringAndFnMatcher`

Reply:
implemented.